### DISCUSSION TITLE
Write file with node fs driver (unstorage)

### DISCUSSION THREAD

**Discussion by woldtwerk** [nuxt/nuxt#24431] [Questions]
How do I write file with unstorage fs driver in production?

In my nuxt.config I added the driver:

```ts
 nitro: {
    storage: {
      content: {
        driver: 'fs',
        base: './content',
      },
    },
},
```

In dev mode reading and writing works fine. When I run the build server I can only read files.

```
entry.877d6e38.js:1 TypeError: Cannot read properties of undefined (reading 'ctx')
```

many thx.

**Comment by pi0** [comment]
Can you please share your full repo? BTW you can benefit from `useStorage('data')` it is preconfigured with fs mounts and works on node in production!

**Reply by woldtwerk** [reply]
good to know. thx.

I just made a small reproduction. No error. yikes.

**Reply by woldtwerk** [reply]
works now. Was something unrelated to unstorage. But this still helped me. Thank you.
